### PR TITLE
[NOMERGE] lxcfs: Fix memory leaks

### PR DESCRIPTION
**Pull request by ndowens**
Created patch from https://github.com/lxc/lxcfs/commit/832904c17476619af8fa74dd25b5f2dd9c52d095

cherry-pick did not work, seems there was too many changes between current version and point of master for fix to apply cleanly if used

### CODE REVIEW

File: srcpkgs/lxcfs-devel [hunk 1]
```diff
@@ -0,0 +1 @@
+lxcfs/
```
Comment (CameronNemo) [inline srcpkgs/lxcfs-devel]: remove the trailing slash

File: srcpkgs/lxcfs/template [hunk 2]
```diff
@@ -14,6 +14,14 @@ checksum=890aa30d960d9b1e53b0c0712bf645c1f1924f750e32cd090f368c1338bd462f
 
 make_dirs="/var/lib/lxcfs 0755 root root"
 
+lxcfs-devel_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove usr/lib/lxcfs/*.so
```
Comment (CameronNemo) [inline srcpkgs/lxcfs/template]: quote wildcards when using vmove, vinstall, vcopy, etc

File: srcpkgs/lxcfs-devel (hunk 1 above)
Comment (ndowens) [reply srcpkgs/lxcfs-devel]: Gonna try and test and see if it does. I didn't know anything about valgrind, so thanks for pointing that out. 

How do I remove the trailing slash on a symlink'd dir?

Comment (CameronNemo) [reply srcpkgs/lxcfs-devel]: Just don't include it when making the symlink. Your shell might autocomplete it but just put lxcfs.

Comment (ndowens) [reply srcpkgs/lxcfs-devel]: @CameronNemo https://paste.pound-python.org/show/ZibDAQWOkG58BfuIPVF1/ 
If I am not mis-reading it, looks like one may be left ?

**Comment by ndowens** [comment]
https://paste.pound-python.org/show/0hS2Ql65CBkOtJIjBFbF/ Not sure that the patch fixes it :\

**Comment by Cogitri** [comment]
A 2K loc patch... can't we just wait for a new upstream release?

**Comment by ndowens** [comment]
Wait for upstream release

**Pull request closed without merging**